rasa_nlu/featurizers/regex_featurizer.py

```diff
@@ -87,6 +102,27 @@ def features_for_patterns(self, message):
         found = [1.0 if m is not None else 0.0 for m in matches]
         return np.array(found)
 
+    def _generate_lookup_regex(self, lookup_table):
+        """creates a regex out of the contents of a lookup table file"""
+        lookup_elements = lookup_table['elements']
+        elements_to_regex = []
+
+        # if it's a list, it should be the elements directly
+        if isinstance(lookup_elements, list):
+            elements_to_regex = lookup_elements
+
+        # otherwise it's a file path.
+        else:
+            with io.open(lookup_elements, 'r') as f:
+                for line in f:
+                    new_element = line.strip()
+                    if new_element:
+                        elements_to_regex.append(new_element)
+
+        """regex matching elements with word boundaries on either side """
```

Comment:
`#`, no `"""` in the middle of the function

Reply:
fixed